tests/src/main/java/testcases/KT006_scope_factory/Scope.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018-2019 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package testcases.KT006_scope_factory
+
+import motif.Creatable
+
+@motif.Scope
+interface Scope : Creatable<Scope.Dependencies> {
+    fun string(): String
```

Comment:
Reading through this code since I was adding a KT006 in an unrelated pull request and feel bad for potentially creating merge conflicts.

I think the test will fail if we remove the string access method - as [this neighborhood in ResolvedGraph](https://github.com/uber/motif/blob/main/core/src/main/kotlin/motif/core/ResolvedGraph.kt#L102-L106) ensures a sink can come from either access methods or factory methods, but _seemingly_ not creatable dependencies. 

There's probably an open question on "whether explicitly declared dependencies from creatable interfaces alone should be sinks, even when they do not satisfy any sources".

Reply:
Recommend closing in favor of #258